hardhat/utils/accounts.ts

```diff
@@ -84,3 +84,22 @@ export const getAccount = (name: string) => {
   }
   return account
 }
+export const getAllAddresses = () => {
+  return ACCOUNTS.reduce<Record<string, string>>((all, acc) => {
+    all[acc.name] = acc.address
+    return all
+  }, {})
+}
+export const getAllKeys = () => ACCOUNTS.map(acc => acc.privateKey)
+export const setAccount = (acc: Account) => {
```

Comment:
Do we need to find the account? 

If you do:

```js
NAMED_ACCOUNTS[acc.name] = acc;
ACCOUNT_ADDRESSES[acc.name] = acc.address;
```

Then the mapped object will hold the updated values